app/templates/packages/project/package.json.ejs

```diff
@@ -16,6 +16,11 @@
 <%_ } else { -%>
     "start:dev": "run-s translate vite:dev",
     "build": "run-s clean:build build:code",
+<%_ } -%>
+<%_ if (hasApollo) { -%>
+    "graphql:schema": "graphql-codegen --config codegen.yml",
+    "graphql:types": "apollo client:codegen --target=typescript --addTypename",
+    "graphql:build": "yarn graphql:schema && yarn graphql:types",
```

Comment:
you can use "npm-run-all"
`"graphql:build": "run-s graphql:schema graphql:types",`

Reply:
&& makes commands sequentially but if `run-s` is used along the project I can change it.